### PR TITLE
ITEP-28898 project name edit field for project import 

### PR DESCRIPTION
**Pull request by ActiveChooN**
## 📝 Description

This PR added an edit field for the project name in the import project modal.

image

JIRA: ITEP-28898

## ✨ Type of Change

Select the type of change your PR introduces:

- [ ] 🐞 **Bug fix** – Non-breaking change which fixes an issue
- [ ] 🚀 **New feature** – Non-breaking change which adds functionality
- [x] 🔨 **Refactor** – Non-breaking change which refactors the code base
- [ ] 💥 **Breaking change** – Changes that break existing functionality
- [ ] 📚 **Documentation update**
- [ ] 🔒 **Security update**
- [ ] 🧪 **Tests**

## 🧪 Testing Scenarios

Describe how the changes were tested and how reviewers can test them too:

- [ ] ✅ Tested manually
- [ ] 🤖 Run automated end-to-end tests

## ✅ Checklist

Before submitting the PR, ensure the following:

- [ ] 🔍 PR title is clear and descriptive
- [ ] 📝 For internal contributors: If applicable, include the JIRA ticket number (e.g., ITEP-123456) in the PR **title**. Do **not** include full URLs
- [ ] 💬 I have commented my code, especially in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [ ] ✅ I have added tests that prove my fix is effective or my feature works

### CODE REVIEW

File: web_ui/src/pages/create-project/create-project-button/create-project-menu.component.tsx [hunk 1]
```diff
@@ -124,10 +138,11 @@ export const CreateProjectMenu = ({ openImportDatasetDialog }: CreateProjectMenu
                                             // in future versions
                                             skipSignatureVerification: true,
                                             keepOriginalDates: false,
-                                            projectName: '',
+                                            projectName: projectName.trim(),
                                         }}
                                         onImportProject={() => {
```
Comment (jpggvilaca) [inline web_ui/src/pages/create-project/create-project-button/create-project-menu.component.tsx]:
```suggestion
                                        onImportProject={dismissImportProject}
```

**Comment by jpggvilaca** [comment]
LGTM! Can we add at least some unit tests for this? (empty name, no name, normal name)

**Comment by pplaskie** [comment]
Would be great if user will be able to see the name of exported project after uploading the file and then edit to be suitable for him/her. But I'm guessing that it's not doable

File: web_ui/src/pages/create-project/create-project-button/create-project-menu.component.tsx (hunk 1 above)
Comment (ActiveChooN) [reply web_ui/src/pages/create-project/create-project-button/create-project-menu.component.tsx]: Fixed

**Comment by ActiveChooN** [comment]
> LGTM! Can we add at least some unit tests for this? (empty name, no name, normal name)

Added some tests

> Would be great if user will be able to see the name of exported project after uploading the file and then edit to be suitable for him/her. But I'm guessing that it's not doable

Indeed, unfortunately, it's not possible. The project parsing is completed after the name is submitted.

**Review by jpggvilaca — APPROVED** [review]

File: web_ui/src/pages/create-project/create-project-button/create-project-menu.test.tsx [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+                expect(screen.getByRole('menuitem', { name: CreateProjectMenuActions.IMPORT_DATASET })).toBeVisible();
+                expect(screen.getByRole('menuitem', { name: CreateProjectMenuActions.IMPORT_PROJECT })).toBeVisible();
+            });
+        });
+
+        it('should open import project dialog when "Create from exported project" is clicked', async () => {
+            await renderCreateProjectMenu();
+
+            const menuButton = screen.getByRole('button', { name: 'Create project menu' });
+            fireEvent.click(menuButton);
```
Comment (pplaskie) [inline web_ui/src/pages/create-project/create-project-button/create-project-menu.test.tsx:85]: I don't remember - did we agree to use fireEvent or userEvent ?

**Review by pplaskie — APPROVED** [review]

File: web_ui/src/pages/create-project/create-project-button/create-project-menu.test.tsx (hunk 2 above)
Comment (pplaskie) [reply web_ui/src/pages/create-project/create-project-button/create-project-menu.test.tsx:85]: IMO can be merged either way, we can clarify and make changes later because we use both ways in the codebase.
